bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/FragmentContainerActivity.kt

```diff
@@ -35,16 +36,25 @@ class FragmentContainerActivity : BridgeBaseActivity() {
                 data.fragmentClass.name,
                 data.fragmentBundle
         )
-        supportFragmentManager
-                .beginTransaction()
-                .replace(R.id.container, fragment)
-                .commit()
+        navigateTo(fragment, false)
     }
 
     override fun onOptionsItemSelected(item: MenuItem) = handleHomeAsBack(item) {
         super.onOptionsItemSelected(item)
     }
 
+    override fun navigateTo(fragment: Fragment, addToBackstack: Boolean) {
+        val transaction = supportFragmentManager
+                .beginTransaction()
+                .replace(R.id.container, fragment)
+
+        if (addToBackstack) {
+            transaction.addToBackStack(null)
+        }
+
+        transaction.commit()
```

Comment:
If you wanted to do this as a single chain I think you could do something like

```kotlin
supportFragmentManager
        .beginTransaction()
        .apply {
            replace(R.id.container, fragment)
            if (addToBackstack) addToBackStack(null)
        }
        .commit()
```

Reply:
That definitely looks cleaner, changed.